packages/neuron-ui/src/components/HistoryDetailPage/index.tsx

```diff
@@ -1,9 +1,10 @@
 import React, { useCallback, useEffect, useMemo, useState } from 'react'
 import { useNavigate, useParams } from 'react-router-dom'
 import { useTranslation } from 'react-i18next'
-import { scriptToAddress } from '@nervosnetwork/ckb-sdk-utils'
+import { scriptToAddress } from 'utils/scriptAndAddress'
 import { calculateUnlockDaoMaximumWithdraw, getTransaction } from 'services/remote'
 import { showPageNotice, transactionState, useDispatch, useState as useGlobalState } from 'states'
+import { CKBComponents } from '@ckb-lumos/rpc/lib/types/api'
```

Comment:
Annotate `CKBComponent` as `type CKBComponent` if it's introduced as a type

Reply:
Added for all.